python/assign.py

```
@@ -26,6 +26,20 @@ def authenticate(token, uid):
 
 
+@app.before_request
+def before_request():
+    if request.method == "OPTIONS":
+        return "", 200
+    app_check_token = request.headers.get("X-Firebase-AppCheck", default="")
+    try:
+        _ = app_check.verify_token(app_check_token)
+        # If verify_token() succeeds, okay to continue to route handler.
+
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)